common/ts/sonarqube/__tests__/TaskReport-test.ts

```diff
@@ -79,5 +79,10 @@ it('should find report files', async () => {
   expect(tl.getVariable).toBeCalledWith('Agent.BuildDirectory');
 
   expect(tl.findMatch).toHaveBeenCalledTimes(1);
-  expect(tl.findMatch).toBeCalledWith('mock root search path', '**\\report-task.txt');
+  expect(findMatchMock.mock.calls[0][0]).toBe('mock root search path');
+
+  // Match using a regular expression to take account of the different
+  // path separators used by "join" on Windows/non-Windows
+  const regEx = new RegExp(/^\*\*[\\/]report-task.txt$/);
```

Comment:
I'm not sure it's better but you could also use [path.join](https://nodejs.org/api/path.html#path_path_join_paths)

Reply:
Doh! That's definitely simpler - not sure why I started messing about with regular expressions...